master/getting-started/kubernetes/installation/hosted/kubeadm/1.6/calico.yaml

```diff
@@ -54,12 +54,19 @@ spec:
       labels:
         k8s-app: calico-etcd
       annotations:
+        # Mark this pod as a critical add-on; when enabled, the critical add-on scheduler
+        # reserves resources for critical add-on pods so that they can be rescheduled after
+        # a failure.  This annotation works in tandem with the toleration below.
         scheduler.alpha.kubernetes.io/critical-pod: ''
```

Comment:
Is the annotation still needed for 1.6? I would imagine it only needs the actual toleration (and I'd like to start cutting out the cruft that is leftover toleration-by-annotations in our 1.6 manifests)